includes/billing.php

```diff
@@ -123,6 +123,21 @@ function getLastMeasurement($bill_id)
     return ($return);
 }//end getLastMeasurement()
 
+function get95thagg($bill_id, $datefrom, $dateto)
+{
+    $mq_sql           = "SELECT count(delta) FROM bill_data WHERE bill_id = '".mres($bill_id)."'";
```

Comment:
mres() is a do nothing function, please use parameterized queries;
like `dbFetchRows('SELECT * FROM devices WHERE device_id=?', [$device_id])`